packages/functional-tests/tests/settings/recoveryPhone.spec.ts

```diff
@@ -366,7 +366,7 @@ test.describe('severity-1 #smoke', () => {
       await signinRecoveryPhone.clickConfirm();
 
       await expect(
-        page.getByText('Invalid or expired confirmation code')
+        page.getByText(/The code is invalid or expired./)
```

Comment:
Any reason to use a regex here even though it's an exact match?

Reply:
Ah right, I must have been thinking of a `getByRole` vs `getByText`. `getByRole` would have failed if not regex because of the link that follows.